doc/ref_cert/lfn/chapters/chapter08.md

```diff
@@ -5,16 +5,175 @@
+| tempest_scenario           | General            |
+| neutron-tempest-plugin-api | Neutron            |
+| patrole                    | Patrole            |
+| barbican                   | Barbican           |
+| networking-bgpvpn          | Networking BGP VPN |
+| networking-sfc             | Networking SFC     |
+
+To complete functional testing, Functest also integrates a few
+[performance tools](https://docs.openstack.org/developer/performance-docs/methodologies/tools.html)
+(2-3 hours) as proposed by OpenStack:
+
+| Testcases  | Benchmarking                |
+| :--------- | :-------------------------- |
+| rally_full | Control Plane (API) testing |
+| rally_jobs | Control Plane (API) testing |
+| vmtp       | Data Plane testing          |
+| shaker     | Data Plane testing          |
+
+And VNFs automatically deployed and tested :
+
+| Testcases    | Benchmarking                        |
+| :----------- | :---------------------------------- |
+| cloudify     | Cloudify deployment                 |
+| cloudify_ims | Clearwater IMS deployed via Coudify |
+| heat_ims     | Clearwater IMS deployed via Heat    |
+| vyos_vrouter | VyOS deployed via Cloudify          |
+| juju_epc     | OAI deployed via Juju               |
+
+Functest should be considered as a whole as it meets multiple objectives about
+the reference implementation:
+- verify all APIs (services, advances, features, etc.) exposed by the reference
+  implementation
+- compare the reference implementation and local deployments from a functional
+  standpoint and from OpenStack control plane and dataplane capabilities
 
+It's worth mentioning that Functest already takes into account the first CNTT
+[profiles](https://git.opnfv.org/functest/tree/functest/ci/config_patch.yaml#n2).
+CNTT should simply add the next Functest inputs according the reference
+implementation:
+- [Functest inputs](https://github.com/opnfv/functest/blob/stable/iruya/functest/utils/env.py#L17)
+- [tempest specific configuration](https://github.com/opnfv/functest/blob/stable/iruya/functest/opnfv_tests/openstack/tempest/custom_tests/tempest_conf.yaml)
+
+Additional links:
+- [Homepage](https://functest.readthedocs.io/en/stable-iruya/)
+- [Run Alpine Functest containers (Iruya)](https://wiki.opnfv.org/pages/viewpage.action?pageId=35291769)
+- [Deploy your own Functest CI/CD toolchains](https://wiki.opnfv.org/pages/viewpage.action?pageId=32015004)
+- [Functest gates](https://build.opnfv.org/ci/view/functest/)
+
+### 8.2.2 Yardstick
+
+### 8.2.3 Bottlenecks
+
+
+### 8.2.4 Test Tools
+1. Shaker:  https://pyshaker.readthedocs.io/en/latest/ (The distributed data-plane testing tool built for OpenStack)
+2. Sonubuoy: https://sonobuoy.io/ It is a diagnostic tool that makes it easier to understand the state of a Kubernetes cluster by running a set of plugins (including Kubernetes conformance tests) in an accessible and non-destructive manner.
+
+### 8.2.5 Scenario Descriptor File (SDF)
```

Comment:
The SDF is not a test framework and should be handled in a separate section, i.e. not be subsection of 8.2.

Reply:
good point. I renamed the title to Relevant Community Projects and Initiatives to be more generic

Reply:
will mark as resolved.